src/scripts/main.js

```diff
@@ -1,3 +1,276 @@
 'use strict';
 
-// write your code here
+const root = document.querySelector('.container');
+
+const allRowsField = root.querySelectorAll('tbody tr');
+const btnStart = root.querySelector('.start');
+const messageFooter = root.querySelector('.message-container');
+const gameScore = root.querySelector('.game-score');
+const widthField = allRowsField[0].children.length;
+
+let field = [];
+
+const createarrField = (arr, column) => {
+  for (let m = 0; m < column; m++) {
+    arr.push([]);
+
+    for (let i = 0; i < widthField; i++) {
+      arr[m].push([]);
+
+      for (let count = 0; count < widthField; count++) {
+        arr[m][i].push('');
+      }
+    }
+  }
+};
+
+const updateHtml = () => {
+  localStorage.setItem('fieldGame2048', JSON.stringify({field:field, score: gameScore.textContent,}));
```

Comment:
```suggestion
  localStorage.setItem('fieldGame2048', JSON.stringify(fields:fields, score: gameScore.textContent,}));
```
якщо це масив то потрібно називати в множинні